lib/vdc/DataMgr.cpp

```diff
@@ -545,6 +545,13 @@ bool is_int(std::string str) {
 	return str.find_first_not_of("0123456789") == std::string::npos;
 }
 
+template <class T>
+void _sanitizeFloats(T *buffer, size_t n) {
+	for (size_t i=0; i<n; i++) {
+		if (isnan(buffer[i])) buffer[i] = INFINITY;
```

Comment:
This line has the following warning message, which doesn't seem very benign

````
/home/shaomeng/Git/VAPOR-flow-src/lib/vdc/DataMgr.cpp:551:35: warning: overflow in implicit constant conversion [-Woverflow]
   if (isnan(buffer[i])) buffer[i] = INFINITY;
````